### ISSUE TITLE
colspan and rowspan broken are broken ?

### ISSUE THREAD

**Issue by brignolij** [bpampuch/pdfmake#2799]
Hello,

I found some weird behavior when playing with table and colspan and rowspan

Here a example using the playground (0.2.14)

```
var dd = {
    content: [
        {
            table: {
                widths: [120, 120, 100, '*'],
                body: [
                    // Table header
                    [{text:'Header  a a a a a a a a a a a a a  a a a a a ',rowSpan:2},
                    {text:'Header  a a a a a a a a a a a a a  a a a a a ',rowSpan:2},
                    {text:'TestVal2  a a a ',colSpan:2,rowSpan:1}, ''],
                    
                    // Table rows
                    ['', '', '', 'Value 4'],
                    [{text:'Header  a a a a a a a a a a a a a  a a a a a ',colSpan:2}, '', 'Value 3', 'Value 4'],
                    ['Row 3', 'Sample value', 'Another sample', 'Another sample']
                ]
            }
        }
    ]
};

```

using Colspan:2 near TestVal2 make the display take two row, instead of one. It take effectively 2 columns. Why ?
How can i avoid this weird behavior, is it a bug ?

![colspan_bug](https://github.com/user-attachments/assets/9a4b9e08-5b36-4d38-9ebb-346fffd2b858)

I have big templates in production that are broken because of this issue.

**Comment by brignolij** [comment]
@fe-cj thanks , i hope it will be fixed in the next release.

**Comment by liborm85** [comment]
Fix released in version 0.2.15.
